.changeset/nine-kiwis-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': minor
```

Comment:
```suggestion
'@backstage/core-components': patch
```

Now that this isn't a breaking change based on @freben comment being addressed 👍